layers/+emacs/org/README.org

```diff
@@ -112,6 +114,46 @@ To use this configuration layer: in the main Spacemacs configuration
 file (=~/.spacemacs=), to the existing =dotspacemacs-configuration-layers= list
 add the =org= entry.
 
+** Agenda notifications
+To enable notifications for agenda events, set the variable
+=org-enable-notifications= to =t=.
+
+#+BEGIN_SRC emacs-lisp
+  (setq-default dotspacemacs-configuration-layers '(
+    (org :variables
+         org-enable-notifications t
+         org-start-notification-daemon-on-startup t)))
+#+END_SRC
+
+This will install and configure the [[https://github.com/tkf/org-mode/blob/master/contrib/lisp/org-notify.el][org-notify]] package. Run the
+=org-notify-start= command to start the =org-notify= notification daemon. To
+start the notification daemon automatically on Spacemacs startup, set the
+variable =org-start-notification-daemon-on-startup= to =t= as shown in the code
+snippet above.
+
+The notifications system can be configured via the =org-notify-add= function. An
+explanation for how to use this function can be found in the initial comment of
+the =org-notify.el= file (=SPC h d f org-notify-add RET o a=) and the in the
+function its docstring. The layer pre-configures a type with the name =appt=
+(i.e. appointments), for which an example of its usage is shown in the example
+below.
+
+The =org-notify= package will scan through your =org-agenda-files= for =TODO=
+headlines that include the =DEADLINE:= stamp. Press ~, d d~ to add a deadline
+stamp to a headline. To set a certain notification type for the item set a
+=:NOTIFY:= property where its value should be the name of a type that you have
+added via =org-notify-add= function. If you add a notification type with the
+name =default=, this type is applied to headlines without the =:NOTIFY:=
+property.
+
+#+begin_src org
+** TODO Enable org notifications in Spacemacs
+   DEADLINE: <2021-02-28 Sun 20:51>
+   :PROPERTIES:
+   :NOTIFY: appt
```

Comment:
I have a question about this, does that mean all my todos must be tagged separately? I would expect to have the package pull in all my todos by default to be honest. Can we configure the package like this by default and give people a hint how they can change it to explicitly tagging messages?

Reply:
You will find in the documentation part that this can be achieved by creating a `default` notification type. So I refer to that part (the README file) of this PR.

Reply:
Yeah I have seen it there, however don't you think that most people will just expect all tasks with deadlines to be notified about. I mean a deadline is a tag afterall, so I would expect this as the sensible standard behaviour i.e. creating a default notification type with package loading and document how to disable this. Then giving an example how to configure your own message types and to set them explicitly for messages.

Reply:
Ah good point, on the one hand, I don't expect users to enable this feature without reading the documentation. But indeed it makes sense to rename the "appt" to "default" to make the tagging unnecessary. :flushed: